miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -757,18 +755,14 @@ public ModelAndView setupForm(@PathVariable Long sampleId, @PathVariable Long pr
   @RequestMapping(value = "/bulk/edit/{sampleIds}", method = RequestMethod.GET)
   public ModelAndView editBulkSamples(@PathVariable String sampleIds, ModelMap model) throws IOException {
     try {
-      String[] split = sampleIds.split(",");
-      List<Long> idList = new ArrayList<Long>();
-      for (int i = 0; i < split.length; i++) {
-        idList.add(Long.parseLong(split[i]));
-      }
+      List<Long> idList = getIdsFromString(sampleIds);
       ObjectMapper mapper = new ObjectMapper();
       JSONArray samplesDtos = new JSONArray();
```

Comment:
Can we do this the other way around and make `sampleDtos` a `List<SampleDto>` and use the Jackson mapper?